README.md

```diff
@@ -1,8 +1,11 @@
 <h1 align='center'>:cherries: Cherry in PyTorch</h1>
 
-Reinforcement Learning &amp; other bedtime stories in PyTorch. This repository is [pytorch](https://pytorch.org) implementation of the [Intro RL course](https://github.com/simoninithomas/Deep_reinforcement_learning_Course) & [Advanced RL course](https://github.com/dennybritz/reinforcement-learning). Before training/playing the agent please install the external [dependencies.](https://github.com/moabitcoin/cherry-pytorch#dependencies) The name cherry is derived from Yann Lecun's [statement](https://www.quora.com/What-do-you-think-about-reinforcement-learning-Is-it-the-cherry-on-the-cake-as-Yann-LeCun-puts-it-1) regarding RL being the cherry on the ML [cake.](https://miro.medium.com/max/1200/1*bvMhd_xpVxfJYoKXYp5hug.png)
+Reinforcement Learning &amp; other bedtime stories in PyTorch. This repository is [pytorch](https://pytorch.org) implementation of the [Intro RL course](https://github.com/simoninithomas/Deep_reinforcement_learning_Course) & [Advanced RL course](https://github.com/dennybritz/reinforcement-learning). Before training/playing the agent please install the external [dependencies.](https://github.com/moabitcoin/cherry-pytorch#dependencies) The name cherry is derived from Yann Lecun's [statement](https://www.quora.com/What-do-you-think-about-reinforcement-learning-Is-it-the-cherry-on-the-cake-as-Yann-LeCun-puts-it-1) regarding RL being the cherry on the ML [cake.](https://miro.medium.com/max/1200/1*bvMhd_xpVxfJYoKXYp5hug.png).
 
 ## :computer: Installation
+```
+Tested with ubuntu:18.04 only
```

Comment:
Docker instructions missing; maybe change all the conda and manual installation instructions to just using docker?

Reply:
Wanna keep both. :)